WalletWasabi.Fluent/Styles/Themes/Base.axaml

```diff
@@ -36,5 +36,7 @@
     <x:Double x:Key="DataGridRowSelectedBackgroundOpacity">1.0</x:Double>
     <x:Double x:Key="DataGridRowSelectedUnfocusedBackgroundOpacity">1.0</x:Double>
 
+    <x:String x:Key="LabelsWatermarkText">i.e. Alice, Bob, Wasabi Inc, Exchange</x:String>
```

Comment:
```suggestion
    <x:String x:Key="LabelsWatermarkText">i.e. Alice, Bob, Coffee Shop, Exchange</x:String>
```

Reply:
there's no Wasabi Inc...

Reply:
This is just an example of a company (or person or exchange) that can be written here. And to not advertise any other company than us this fantasy name was born.